### DISCUSSION TITLE
Github doesn't accept my enrollment proof document for some reason

### DISCUSSION THREAD

**Discussion by jHONFK** [community/community#140193] [GitHub Education]
### Hi everyone,

![page_1](https://github.com/user-attachments/assets/99a3cb46-af50-44a5-88a5-080603f165e7)

Above, the document that github is not accepting, I have no other proof, this is the official document, it even has the possibility of verification in the footer, what do I do now?

**Comment by Kartikey2011yadav** [comment]
It's just my assumption but i guess github is unable to recognize the contents of the document. For my case i uploaded my college fees submission receipt / Admission letter that worked for me. If you got something like that, i suggest you try it out.
